react_week1/src/todo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Todo = (props) =>{
+    return(
+        <h2>
+        {props.description},
+        {props.deadLine}
+        </h2>
+    );
+
+}
+
+
+const RenderTodo = (props)=> {
+    return(
+        <ol>
+            <h1 className = 'todo-header'>
+               Todo List 
+            </h1>
+        { 
+    props.toDoList.map(todo => <li><Todo description = {todo.description} deadLine = {todo.deadLine}/> </li>)
+    } 
+        </ol>
+    );
+
```

Comment:
maybe work a little with the formatting here. for example:

```
return (
  <ol>
    <h1 className="todo-header">Todo List</h1>
    {props.toDoList.map((todo) => (
      <li>
        <Todo description={todo.description} deadLine={todo.deadLine} />{" "}
      </li>
    ))}
  </ol>
);
```

is a little easier to read.